syscall/linux/constants.lua

```diff
@@ -455,9 +455,19 @@ c.SO = strflag(arch.SO or {
   WIFI_STATUS        = 41,
   PEEK_OFF           = 42,
   NOFCS              = 43,
+  LOCK_FILTER        = 44,
+  SELECT_ERR_QUEUE   = 45,
+  BUSY_POLL          = 46,
+  MAX_PACING_RATE    = 47,
+  BPF_EXTENSIONS     = 48,
+  INCOMING_CPU       = 49,
+  ATTACH_BPF         = 50,
+  ATTACH_REUSEPORT_CBPF = 51,
+  ATTACH_REUSEPORT_EBPF = 52,
 })
 
 c.SO.GET_FILTER = c.SO.ATTACH_FILTER
+c.SO.DETACH_BPF = c.SO.SO_DETACH_FILTER
```

Comment:
It's an alias for the same option http://lxr.free-electrons.com/source/include/uapi/asm-generic/socket.h?v=4.0#L88